web/src/beta/features/Editor/tabs/map/SidePanel/index.tsx

```diff
@@ -0,0 +1,67 @@
+import Loading from "@reearth/beta/components/Loading";
+import SidePanelSectionField from "@reearth/beta/components/SidePanelSectionField";
+import Text from "@reearth/beta/components/Text";
+import SidePanelCommon from "@reearth/beta/features/Editor/SidePanel";
+import { useSceneFetcher } from "@reearth/services/api";
+import { PropertySchemaGroup } from "@reearth/services/gql";
+import { useT } from "@reearth/services/i18n";
+import { useTheme, styled } from "@reearth/services/theme";
+
+type Props = {
+  sceneId: string;
+};
+
+type GroupSectionFieldProps = {
+  groups: PropertySchemaGroup[];
+};
+
+const GroupSectionField: React.FC<GroupSectionFieldProps> = ({ groups }) => {
+  const t = useT();
+
+  return (
+    <SidePanelSectionField title={t("Scene")}>
+      {groups.map(({ schemaGroupId, title }) => (
+        <GroupSectionFieldText key={schemaGroupId} size="footnote">
+          {t(title as string)}
```

Comment:
You don't need to wrap the title in i18n. The title we get from the backend will be based on our language settings, so just show as is (and without the type assertion) 🙏🏻